guide/moderator/lock.md

```diff
@@ -0,0 +1,11 @@
+## User Handbook
+
+## Locking a Discussion
+
+To lock a discussion, as always you'll first need to find the discussion in threaded view, once you have go ahead and bring up the thread menu by hovering the mouse over the right of the thread, and selecting the three dots menu, as show in the screenshot below:
+
+![SS - Showing thread menu](687474703a2f2f692e696d6775722e636f6d2f4558665258415a2e706e67.png)
+
+With the menu, click on the **Lock** label which will immediately lock the thread, preventing any futher replies. A confirmation of the thread lock will be a padlock over the user's avatar as shown below:
```

Comment:
It will allow replies from users who can lock discussions. Not sure if you need to mention this or not.